src/containers/shared/components/Transaction/XChainCreateBridge/Simple.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import Account from '../../Account'
+import { TransactionSimpleComponent, TransactionSimpleProps } from '../types'
+import { SimpleRow } from '../SimpleRow'
+
+export const Simple: TransactionSimpleComponent = (
+  props: TransactionSimpleProps,
+) => {
+  const { t } = useTranslation()
+  const { data } = props
+  const {
+    instructions: {
+      lockingDoor,
+      lockingIssue,
+      issuingDoor,
+      issuingIssue,
+      signatureReward,
+      minAccountCreateAmount,
+    },
+  } = data
```

Comment:
oh sorry the point here - if you use this, is to get rid of `const { data } = props`. there's another nesting so it would either be:

```javascript
const {
  data: {
    instructions: {
      ...
} = props
```

or 

```javascript
const {
   ...
} = props.data.instructions
```

Reply:
Ah ok, I was confused why it only changed part of it.